packages/cms-migration/src/news-migration.ts

```diff
@@ -0,0 +1,103 @@
+import assert from 'assert';
+import * as contentful from 'contentful-management';
+import { parseHtml } from 'contentful-html-rich-text-converter';
+
+import {
+  getAccessTokenFactory,
+  SquidexRest,
+  RestNews,
+} from '@asap-hub/squidex';
+
+const migrateNews = async () => {
+  [
+    'CONTENTFUL_MANAGEMENT_ACCESS_TOKEN',
+    'CONTENTFUL_SPACE_ID',
+    'CONTENTFUL_ENV_ID',
+    'CRN_SQUIDEX_APP_NAME',
+    'CRN_SQUIDEX_CLIENT_ID',
+    'CRN_SQUIDEX_CLIENT_SECRET',
+    'SQUIDEX_BASE_URL',
+  ].forEach((env) => {
+    assert.ok(process.env[env], `${env} not defined`);
+  });
+
+  const {
+    CONTENTFUL_MANAGEMENT_ACCESS_TOKEN,
+    CONTENTFUL_SPACE_ID,
+    CONTENTFUL_ENV_ID,
+    CRN_SQUIDEX_APP_NAME,
+    CRN_SQUIDEX_CLIENT_ID,
+    CRN_SQUIDEX_CLIENT_SECRET,
+    SQUIDEX_BASE_URL,
+  } = process.env;
+
+  const getAuthToken = getAccessTokenFactory({
+    clientId: CRN_SQUIDEX_CLIENT_ID!,
+    clientSecret: CRN_SQUIDEX_CLIENT_SECRET!,
+    baseUrl: SQUIDEX_BASE_URL!,
+  });
+
+  const squidexNewsClient = new SquidexRest<RestNews>(
+    getAuthToken,
+    'news-and-events',
+    { appName: CRN_SQUIDEX_APP_NAME!, baseUrl: SQUIDEX_BASE_URL! },
+  );
+
+  const contentfulClient = contentful.createClient({
+    accessToken: CONTENTFUL_MANAGEMENT_ACCESS_TOKEN!,
+  });
+
+  const contentfulSpace = await contentfulClient.getSpace(CONTENTFUL_SPACE_ID!);
+  const contentfulEnvironment = await contentfulSpace.getEnvironment(
+    CONTENTFUL_ENV_ID!,
+  );
+
+  const squidexNews = await squidexNewsClient.fetch();
+
+  const newsFields = [
+    'title',
+    'frequency',
+    'link',
+    'linkText',
+    'shortText',
+    // 'thumbnail', // TODO: deal with thumbnail
+  ];
+
+  const firstNews = squidexNews.items[0];
+
+  [firstNews!].forEach(async ({ data, id }) => {
```

Comment:
obvs this works for one there is just one and i guess we could still have dome processed one by one although since we are sending them along with their IDs than means they dont need to be inserted in a correct order and therefore we could insert like 10 at once or something. First I would see if the client supports batches and second I would maybe consider sending out more than 10 requests in parallel.

Now that I think about it - for News all this makes no sense cause we have like 10 news items in total, maybe something for us to think about for RO migration